axelar-chains-config/info/stagenet.json

```diff
@@ -2031,6 +2105,21 @@
           "domainSeparator": "0x254a1ae90417f8649798698e95a1140808a701c38c8f5723a930b9d0666e54b4",
           "address": "axelar1ktjfpfrlgmh8s76ltjhhv0xcj0st0d0xrjh5rvm3gr5u66x9xvpsm0juw8"
         },
+        "stellar": {
```

Comment:
wrong chain name, should have been `stellar-2024-q4`, please follow the release doc carefully. we can let this one through, but the next addition should use the correct naming

Reply:
Oh man ...